packages/okta-react-native/android/src/main/java/com/oktareactnative/OktaSdkBridgeModule.java

```diff
@@ -347,6 +347,22 @@ public void onError(String e, AuthorizationException error) {
         }
     }
 
+    @ReactMethod
+    public void clearTokens(final Promise promise) {
+        try {
+            if (webClient == null || authClient == null) {
```

Comment:
`if (webClient == null || authClient == null)` should be `if (webClient == null && authClient == null)`